near-sdk/src/store/lazy/mod.rs

```diff
@@ -66,7 +66,7 @@ where
 {
     /// Key bytes to index the contract's storage.
     storage_key: Box<[u8]>,
-    #[borsh_skip]
+    #[borsh(skip, bound(deserialize = ""))]
     /// Cached value which is lazily loaded and deserialized from storage.
     cache: OnceCell<CacheEntry<T>>,
```

Comment:
`OnceCell` has a `Default` [implementation](https://docs.rs/once_cell/1.18.0/src/once_cell/lib.rs.html#425) of an empty cell, not requiring `Default` on the contained type.